components/ui/Navbar.js

```diff
@@ -35,25 +37,37 @@ export const Navbar = () => {
           {/* TASK - React 1 week 2 */}
           {/* Create a <NavItem> component, which accepts the following:  */}
           {/* title, link, isActive  */}
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[0].link === currentPath,
-          })}>
-            <Link href={navbarItems[0].link}><b>01</b> {navbarItems[0].title}</Link>
+          <li
```

Comment:
Since you are repeating yourself, this could be an array with a `.map()` function

Reply:
Noted, But it was week-3 assignment to replace the repeating code with map(). I will do that in next home work submission